assets/scss/core/_variables.scss

```diff
@@ -14,7 +14,7 @@ $dp-font-small-lineheight: 18px;
 $dp-font-weight: 400 !default;
 $dp-font-family: Arial, sans-serif !default;
 $dp-font-family-proximanova: "proxima-nova", Helvetica;
-$dp-font-size: 16px !default;
+$dp-font-size: 15px !default;
```

Comment:
Is there not a font size level?

Reply:
This case is an exception. The levels we have jump from 6 to 6. 12 is very small and 18 is too big. The mockups have the size in 15. I preferred to adjust to the design. In any case, this size is within a variable and can be modified.

Reply:
I is fine for me, thanks for the clarification!